Improve user guide by checking commands on a freshly installed machine

adamschmideg opened the pull request.

adamschmideg committed:
Add command to install aws-iam-authenticator

adamschmideg committed:
More specific url to helm install

adamschmideg committed:
Required version of aws-cli

adamschmideg committed:
Details how to configure aws

adamschmideg committed:
Ensure we're in the right directory

adamschmideg committed:
Replace the content of your-values.yaml with a link to the file

adamschmideg committed:
Reorder helm commands so they can all executed in a script with tear down at the end

nonsense commented:
There are references to our private repo and mistakes in the updates, as well as changes to the styling that are not necessary - for example multi-line to single-line commands.

Also titles/headings that make no improvement to the documentation, such as `Make sure you're in the right kubernetes repository`. Therefore I am not accepting this PR.

adamschmideg commented:
I learned I should make a PR against https://github.com/ethereum/swarm-cluster
I changed multi-line commands into single-line ones, because it's easier to copy them into the console and edit the to suit my needs.
I added "Make sure you're in the right kubernetes repository", because the user guide didn't tell I have to check out a repository and which one. I ended up in the wrong repo and couldn't file the config files the document is referring to.
If you disagree with other changes, please be specific which ones.
